src/Aks/Aks/help/Invoke-AzAksRunCommand.md

```diff
@@ -0,0 +1,261 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Aks.dll-Help.xml
+Module Name: Az.Aks
+online version: https://docs.microsoft.com/powershell/module/az.aks/invoke-azaksruncommand
+schema: 2.0.0
+---
+
+# Invoke-AzAksRunCommand
+
+## SYNOPSIS
+Run a shell command (with kubectl, helm) on your aks cluster, support attaching files as well.
+
+## SYNTAX
+
+### GroupNameParameterSet (Default)
+```
+Invoke-AzAksRunCommand [-ResourceGroupName] <String> [-Name] <String> -CommandString <String>
+ [-CommandFile <String[]>] [-AsJob] [-Force] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [-SubscriptionId <String>] [<CommonParameters>]
```

Comment:
I don't think we need to append type to parameter name, so maybe `-Command` is enough than `-CommandString`?

Also the name `-CommandFile` gives me the impression that I can write the command into a file and let Aks run it, but that's not true. How about we call it "-CommandAttachment" or "-CommandAttachmentFile" or "-AttachmentFile"?